central/cve/image/csv/handler.go

```diff
@@ -111,14 +111,14 @@ func ImageCVECSVHandler() http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		query, rQuery, err := parser.ParseURLQuery(r.URL.Query())
 		if err != nil {
-			csv.WriteError(w, http.StatusBadRequest, err)
+			csv.WriteErrorWithCode(w, http.StatusBadRequest, err)
```

Comment:
Our sentinel `errox.InvalidArgs` maps to `http.StatusBadRequest`. Do we want to say:
```suggestion
			csv.WriteError(w, errox.InvalidArgs.CausedBy(err))
```
to add "invalid arguments: " prefix to the error parser message?

Reply:
I guess for me the biggest value of sentinels is not that extra "invalid arguments: " prefix but error class determination at a point where it's known best. However, your suggestion might help get rid of `WriteErrorWithCode()` and hence increase the visibility of sentinels.